src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -3607,6 +3603,11 @@ public enum RisingRivals implements LogicCardInfo {
               checkFaint()
             }
             shuffleDeck()
+            delayed {
+              before BETWEEN_TURNS
+              new CardList(thisCard).moveTo(thisCard.player.owner.pbg.hand)
+              unregisterAfter 1
+            }
```

Comment:
What is this?

Reply:
This is what happens when you blow past the Ballmer peak